images/scripts/opensuse-tumbleweed.setup

```diff
@@ -0,0 +1 @@
+opensuse.setup
```

Comment:
Let's not do this indirection just yet -- if we get a second similar suse image, we can do the factorization then. The micro image setup will most likely look completely different.